mirny_cpld.py

```diff
@@ -17,7 +17,11 @@
         # high: type 2 receiver, failsafe low
         ("fsen", 0, Pins("P80")),
 
-        ("mezz_io", 0, Pins("P57 P58 P59 P60 P61 P64 P68 P69")),
+        #almazny            CLK MOSI LT1 LT2  LT3 LT4 CLR
+        #                   IO8  11  10  13   15  0     3
+        ("mezz_io", 0, Pins("P94 P97 P96 P100 P92 P57 P60")),
+        #almazny                NOE1 NOE2 NOE3  4
+        ("almazny_noe", 0, Pins("P95 P98 P101 P58")),
```

Comment:
The comment is incomplete don't reflect the schematics and the changes are needlessly forward-incompatible.